crates/core/component/auction/src/auction/dutch.rs

```diff
@@ -65,6 +69,28 @@ pub struct DutchAuctionDescription {
     pub nonce: [u8; 32],
 }
 
+impl DutchAuctionDescription {
+    /// Compute the unique identifier for the auction description.
+    pub fn id(&self) -> AuctionId {
+        let mut state = blake2b_simd::Params::default()
+            .personal(DUTCH_AUCTION_DOMAIN_SEP)
```

Comment:
I think the personalization bytes should be maximum 16 bytes: https://docs.rs/blake2b_simd/latest/blake2b_simd/struct.Params.html#method.personal

Reply:
oh great catch